Misc/NEWS.d/next/Library/2024-09-24-18-16-59.gh-issue-58956.0wFrBR.rst

```diff
@@ -0,0 +1 @@
+Setting a :mod:`pdb` breakpoint on a function that's already in the call stack will enable the trace function on that frame so the breakpoint would trigger.
```

Comment:
This can probably be reworded to make it clearer what the change is (e.g., "Fix bug where..."). They way it's written it's not immediately clear what the old vs new behaviour is.

Reply:
this ^

Reply:
Ah, missed that. Do you think the current news is better?